google/ggrc-core
# GGRC-5298 Update attrs for issue in GGRC when issue attrs have been changed in issue tracker

# Issue description

Update attrs for issue in GGRC when issue attrs have been changed in issue tracker

# Steps to test the changes

* Create Issue with enabled sync in GGRC;
* Modify assignee or verifier in Issue tracker;
* Run issue tracker job manually;
* Emails should be updated in GGRC.

# Solution description

Cron job for issue attributes sync has been implemented. 
Also I have implemented integration test for this functionality.

# Sanity checklist




# PR Review checklist
